src/layout/Tabs/TabsTheme.ts

```diff
@@ -2,43 +2,71 @@ export interface TabsTheme {
   base: string;
   list: {
     base: string;
+    indicator: string;
+    divider: string;
+    variant: {
+      primary: {
+        indicator: string;
+        divider: string;
+      };
+      secondary: {
+        indicator: string;
+        divider: string;
+      };
+    };
     tab: {
       base: string;
+      button: string;
       selected: string;
       disabled: string;
+      variant: {
+        primary: {
+          button: string;
+        };
+        secondary: {
+          button: string;
+        };
+      };
     };
-    indicator: string;
   };
   panel: string;
 }
 
 const baseTheme: TabsTheme = {
   base: 'flex flex-col',
   list: {
-    base: 'flex text-center flex-wrap -mb-px position-relative',
-    tab: {
-      base: 'rounded-none border-b-2 border-transparent',
-      selected: '',
-      disabled: 'cursor-not-allowed opacity-50'
+    base: 'flex text-center flex-wrap -mb-px',
+    indicator: 'bg-primary absolute bottom-0 left-0 right-0',
+    divider: 'w-full h-px border-0',
+    variant: {
+      primary: {
+        indicator: 'h-[4px]',
+        divider: 'bg-surface'
+      },
+      secondary: {
+        indicator: 'h-[2px]',
+        divider: 'bg-gradient-to-r from-transparent to-transparent via-blue-500'
```

Comment:
`via-blue-500` or `via-primary` ?

Reply:
oh wait.. that's for `secondary` variant

Reply:
I think `via-primary` is still right even though this is the `secondary` variant